s3upload.js

```diff
@@ -7,15 +7,15 @@ S3Upload.prototype.signingUrl = '/sign-s3';
 S3Upload.prototype.fileElement = null;
 S3Upload.prototype.files = null;
 
-S3Upload.prototype.onFinishS3Put = function(signResult) {
+S3Upload.prototype.onFinishS3Put = function(signResult, file) {
     return console.log('base.onFinishS3Put()', signResult.publicUrl);
 };
 
-S3Upload.prototype.onProgress = function(percent, status) {
-    return console.log('base.onProgress()', percent, status);
+S3Upload.prototype.onProgress = function(percent, status, file) {
+    return console.log('base.onProgress()', percent, status, file);
 };
 
-S3Upload.prototype.onError = function(status) {
+S3Upload.prototype.onError = function(status, file) {
     return console.log('base.onError()', status);
 };
```

Comment:
Looks like these methods have been updated, but the actually calling of them hasn't been.  Line 40, 87, etc., need to pass `file` as the third arg it looks like, no?